app/mesh/changelog.md

```diff
@@ -4,6 +4,28 @@ no_search: true
 no_version: true
 ---
 
+## 1.2.6
+
+> Released on 2021/05/13
+
+### Changes
+
+Built on top of [Kuma 1.1.6](https://github.com/kumahq/kuma/blob/master/CHANGELOG.md#116).
+
+- Intermediate Certificate Authorities (CAs) are now supported with Vault integration.
+- You can now specify any and all tags in a Traffic Permission policy for Vault integration.
+- You can now specify TCP and HTTP health checks at the same time in the same policy. The health check policy also 
+now includes a `reuse_connection` option.
+- The `--gateway` flag is now available in the CLI.
+- You can now install Kong Ingress with the CLI.
```

Comment:
To double-check: is it Kong Ingress or the Kong Ingress Controller that they're able to install? Those would be two different things, right? Ie potentially a Kong Ingress CRD or the controller?

Reply:
> To double-check: is it Kong Ingress or the Kong Ingress Controller that they're able to install? Those would be two different things, right? Ie potentially a Kong Ingress CRD or the controller?

KIC, from context. Let's merge as is and fix as a fast follow if other reviewers of Kuma content want more detail.

Reply:
lolnope updated. Kong Gateway, per Marco.